dev-cpp/libmcpp/libmcpp-2.7.2-r3.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
you didn't use repoman for commiting? (it would fix the year)

Reply:
Fixed